src/topic_tree_.toit

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
```

Comment:
Not sure it feels necessary to have this private (filename ends with _).

Reply:
done.